### PR TITLE
add the possibility for hidden password input

### PR DESCRIPTION
**Pull request by dondeli**
Hi
Sometime, hidden password entry can be useful. The code should be reverse compatible.

Regards
Michel

### CODE REVIEW

File: pyark/__init__.py [hunk 1]
```diff
@@ -457,6 +459,10 @@ def main(argv=None):
     ch.setFormatter(log_formatter)
     logger.addHandler(ch)
 
+    if not args.apipassword:
+        args.apipassword = getpass("Please enter password for {}: "
+                                   .format(args.apiuser))
+
```
Comment (karras) [inline pyark/__init__.py]: Please remove the whitespaces on line 465 and make flake8 / Travis green again. :)

**Review by karras — CHANGES_REQUESTED** [review]

**Review by karras — APPROVED** [review]
LGTM

**Comment by karras** [comment]
Thanks!

**Comment by karras** [comment]
@dondeli I've published the new version to PyPi (https://pypi.org/project/pyark/). Have fun!

**Comment by dondeli** [comment]
@karras THX a lot!

**Pull request merged**